test/Microsoft.Framework.CommonTestUtils/TestUtils.cs

```diff
@@ -9,11 +9,18 @@
 using System.Security.Cryptography;
 using System.Text;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Infrastructure;
```

Comment:
Did you forget to do that part during renaming?
https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Framework.Runtime.Abstractions/CallContextServiceLocator.cs#L4